packages/fast-components-styles-msft/src/radio/index.ts

```diff
@@ -122,13 +111,40 @@ const styles: ComponentStyles<RadioClassNameContract, DesignSystem> = {
     },
     radio__checked: {
         "& $radio_stateIndicator": {
-            "&::before": {
-                background: neutralForegroundRest,
-                ...highContrastHighlightBackground,
-            },
+            background: neutralForegroundRest,
+            "box-shadow": format(
+                "inset 0 0 0 {0} {1}",
+                indicatorMargin,
+                neutralFillInputRest
+            ),
+            ...highContrastHighlightBackground,
+            border: format(
+                "{0} solid {1}",
+                toPx<DesignSystem>(outlineWidth),
+                neutralOutlineRest
+            ),
         },
-        "&:hover $radio_stateIndicator::before": {
+        "&:hover $radio_stateIndicator": {
             ...highContrastSelectedBackground,
+            "box-shadow": format(
+                "inset 0 0 0 {0} {1}",
+                indicatorMargin,
+                neutralFillInputHover
+            ),
+        },
+        "&:hover:enabled $radio_stateIndicator": {
```

Comment:
why is this selector necessary? It applies the same box-shadow as above to the same element

Reply:
Ideally we keep the ones with ":enabled" for elements that support this selector. This should be on "active" as well below.